examples/text-to-text/falconsai-text-summarisation/new_pipeline.py

```diff
@@ -0,0 +1,72 @@
+import typing as t
+
+from transformers import pipeline as t_pipeline
```

Comment:
`hf_pipeline` might be more appropiate

Reply:
Cool thanks for that, have commited your suggestions